stellargraph/layer/sort_pooling.py

```diff
@@ -64,7 +64,7 @@ def compute_output_shape(self, input_shapes):
             An input shape tuple.
         """
         if self.flatten_output:
-            return input_shapes[0], self.k * input_shapes[2]
+            return input_shapes[0], self.k * input_shapes[2], 1
```

Comment:
Just so I understand, what's the reason for the extra `1` here?

Reply:
The `Conv1D` layer requires input to be 3D tensor. See https://www.tensorflow.org/api_docs/python/tf/keras/layers/Conv1D